notebooks/NIRSpec_NSClean/FS_NSClean_example.ipynb

```diff
@@ -0,0 +1,1764 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        slice_indx=1,
</pre><p>Another minor issue that caught my attention: The parameter for the slice index is named <code>slice_indx</code>,but in the docstring it's referred as <code>slice_index.`</code>  It's important to keep these consistent to avoid confusion.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Fixed